frontend/src/containers/ConfigStep/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  StyledFeaturedRangePickerContainer,
+  StyledDateRangePickerContainer,
+  StyledDateRangePicker,
+  RemoveButton,
+} from '@src/containers/ConfigStep/DateRangePicker/style';
+import {
+  DEFAULT_SPRINT_INTERVAL_OFFSET_DAYS,
+  REMOVE_BUTTON_TEXT,
+  DATE_RANGE_FORMAT,
+  START_DATE_INVALID_TEXT,
+  END_DATE_INVALID_TEXT,
+} from '@src/constants/resources';
+import {
+  calculateStartDateShouldDisable,
+  calculateEndDateShouldDisable,
+  calculateLastAvailableDate,
+} from '@src/containers/ConfigStep/DateRangePicker/validation';
+import {
+  initDeploymentFrequencySettings,
+  saveUsers,
+  updateShouldGetBoardConfig,
+  updateShouldGetPipelineConfig,
+} from '@src/context/Metrics/metricsSlice';
+import { IRangePickerProps } from '@src/containers/ConfigStep/DateRangePicker/types';
+import { selectDateRange, updateDateRange } from '@src/context/config/configSlice';
+import { useAppDispatch, useAppSelector } from '@src/hooks/useAppDispatch';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import { TextField, TextFieldProps } from '@mui/material';
+import { Z_INDEX } from '@src/constants/commons';
+import { Nullable } from '@src/utils/types';
+import dayjs, { Dayjs } from 'dayjs';
+import { useCallback } from 'react';
+import isNull from 'lodash/isNull';
+
+const HelperTextForStartDate = (props: TextFieldProps) => (
+  <TextField {...props} variant='standard' helperText={props.error ? START_DATE_INVALID_TEXT : ''} />
+);
+
+const HelperTextForEndDate = (props: TextFieldProps) => (
+  <TextField {...props} variant='standard' helperText={props.error ? END_DATE_INVALID_TEXT : ''} />
+);
+
+export const DateRangePicker = ({ startDate, endDate, index }: IRangePickerProps) => {
+  const dispatch = useAppDispatch();
+  const dateRangeGroup = useAppSelector(selectDateRange);
+  const isShowRemoveButton = dateRangeGroup.length > 1;
+  const dateRangeGroupExcludeSelf = dateRangeGroup.filter((_, idx) => idx !== index);
+  const shouldStartDateDisableDate = calculateStartDateShouldDisable.bind(
```

Comment:
rename `shouldStartDateDisableDate` to `isStartDateDisabled`

same as `shouldEndDateDisableDate`

Reply:
ok